extensions/typescript/src/typescriptServiceClient.ts

```diff
@@ -808,6 +826,10 @@ export default class TypeScriptServiceClient implements ITypescriptServiceClient
 				}
 				break;
 		}
+		if (telemetryData.telemetryEventName === 'projectInfo') {
+			this._tsserverVersion = properties['version'];
+		}
+
 		/* __GDPR__
 			"typingsInstalled" : {
```

Comment:
The version field will need to be added to the gdpr annotations as well. 

@kieferrm Would `__GDPR__COMMON__` work for this?

Reply:
__GDPR__COMMON__ means that this is a property that is on every single telemetry event we send. The TS server version is only on TS related events, so it is not a common property.

Reply:
Maybe this can be done with __GDPR__FRAGMENT__ and a wildcard to match on the name? Or would we need to extend the comment-handling functionality?